### ISSUE TITLE
New endpoint: api/school-status?id=school

### ISSUE THREAD

**Issue by lukarenko** [sledilnik/data-api#76]
We would like to have interface, which would return filtered data from these two CSV files into JSON (as two arrays of records):

1. [schools-absences.csv](https://github.com/sledilnik/data/blob/master/csv/schools-absences.csv)
2. [schools-regimes.csv](https://github.com/sledilnik/data/blob/master/csv/schools-regimes.csv)

Suggested format:
```
{
   "school1": {
        "absences:  [
              { absence1 },
              { absence2 }
        ]
        "regimes: [
              { regime1 },
              { regime2 }
        ]
   }
}

```

Bonus point: can we specify multiple school IDs in argument list (api/school-status?id=school1,school2,school3)?

**Comment by MihaMarkic** [comment]
What do you mean by filtered exactly? The usual from/to plus the list of school ids?

**Comment by lukarenko** [comment]
@MihaMarkic filtering with list of IDs is enough for now. 
Maybe we can also support from= filter?

Also: please change `school` from int to string to make it consistent with use in array.

**Comment by MihaMarkic** [comment]
Implemented in 1.11.1
